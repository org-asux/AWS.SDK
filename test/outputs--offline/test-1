__OFFLINE_MODE - AWS.SDK PROJECT - createKeyPairEC2()
