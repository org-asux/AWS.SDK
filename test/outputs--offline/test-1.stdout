org.ASUX.yaml.AWSSDKcreateKeyPairEC2(ap-south-1,testSSHKeyPair2): AWS.SDK library is running in __OFFLINE__ mode.  So this method is a 'NOOP'!!!!!!!!
File /Users/Sarma/.aws/testSSHKeyPair2 created.
